scripts/update.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+ORIGINAL_REPO=iris-dni/iris-frontend.git
+HAS_CONFIGURED_UPSTREAM=$(git config --get remote.upstream.url | grep $ORIGINAL_REPO)
+
+if [ "$HAS_CONFIGURED_UPSTREAM" ]; then
+  read -p "You should definitely stash any changes before doing this. Continue? [Y/n] " -n 1 -r
```

Comment:
Doesn't `[Y/n]` imply that `Y` is the default here and simply hitting return would do the update?